static/js/microcloud-slider.js

```diff
@@ -0,0 +1,79 @@
+// Slider that shows 3 resource items at a time
+function threeItemSlider() {
+  const nextBtn = document.querySelector('.js-slider--next');
+  const prevBtn = document.querySelector('.js-slider--prev');
+  const items = document.querySelector('.js-slider--items').children;
+  let resultsCount = document.getElementById('js-results-count');
```

Comment:
You should null check these elements are found before continuing in case the mark up changes and js crashes.